README.adoc

```diff
@@ -343,6 +344,14 @@ The following table documents the patches we've made to PhantomJS and tracks the
 |{icon-exclamation-dim} todo
 |===
 
+=== WebSlides
+
+PhantomJS doesn't support flexbox very well and WebSlides makes heavy use of flexbox.
```

Comment:
We may want to add a small comment / note that the next rebase to latest PhantomJS version (2.5) will provide proper support for Flexbox (among many other goodness 😃).